pkg/upgrade/repo/manager.go

```diff
@@ -0,0 +1,95 @@
+package repo
+
+import (
+	"fmt"
+
+	"github.com/weaveworks/pctl/pkg/git"
+)
+
+const main = "main"
+
+//go:generate counterfeiter -o fakes/fake_repo_manager.go . RepoManager
+// RepoManager for managing a git repository
```

Comment:
This should be the other way around. Frigging linter is worth crap. :/

Reply:
I don't follow :thinking:, which part should move?

Reply:
```suggestion
// RepoManager for managing a git repository
//go:generate counterfeiter -o fakes/fake_repo_manager.go . RepoManager
```

Reply:
ohh I see. sure :smile: